.github/workflows/conda.yml

```diff
@@ -0,0 +1,106 @@
+name: Conda
+
+on:
+  pull_request:
```

Comment:
I did this so it would actually run while working on it, this should be dropped before merging

Reply:
```suggestion
```